### DISCUSSION TITLE
apisix stream-route healthcheck

### DISCUSSION THREAD

**Discussion by whioue** [apache/apisix#5238] [Q&A]
Can I get the stream-route health check data through the /V1/healthcheck interface?

**Comment by tokers** [comment]
> Can I get the stream-route health check data through the /V1/healthcheck interface?

It shows the upstream health check status from three dimensions: route (for the upstream embedded inside it), service (for the upstream embedded inside it), and the other is upstream. So for the independent upstream, you can see their health check status without any problems, but for the upstream embedded upstream, you can't, this can be supported to submit a PR.
